### ISSUE TITLE
Onboarding: project character and word statistics

### ISSUE THREAD

**Issue by mmartin9684-sil** [sillsdev/serval#277]
During onboarding, when the user supplies a translation or back translation, it would be helpful to capture some statistics about the text of the project, such as:

- words and word frequencies
- characters and character frequencies
- distribution plots of this word and character information

In addition, after a preprocess/train/test run, it would be helpful to capture some token and word statistics indicating which tokens and words were part of the train, validation, and/or test sets for both the source and target texts.  In particular, flagging any inconsistencies -
 tokens or words in the source / target validation or test set that were not part of the training set - would be helpful.
